libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -228,6 +228,7 @@ class AC_PrecLand
         uint64_t time_usec;
     };
     ObjectArray<inertial_data_frame_s> *_inertial_history;
+    struct inertial_data_frame_s *_inertial_data_delayed;
```

Comment:
I am not sure if I understand the rationale behind this change

Reply:
The next commit requires this change. Doing it here and now makes the next commit a lot easier to understand